lib/tasks/data_import.rake

```diff
@@ -1,10 +1,108 @@
-desc "Import CSV data"
-task :import_data do
-  require 'csv'
-  customers = CSV.read('db/data/customers.csv')
-  invoice_items = CSV.read('db/data/invoice_items.csv')
-  invoices = CSV.read('db/data/invoices.csv')
-  items = CSV.read('db/data/items.csv')
-  merchants = CSV.read('db/data/merchants.csv')
-  transactions = CSV.read('db/data/transactions.csv')
-end
\ No newline at end of file
+require 'csv'
+
+desc 'Import CSV data'
+task import_data: [:environment] do
+  InvoiceItem.destroy_all
```

Comment:
The order to delete these took some trial and error and worked differently in the development and test environments. Will see how well this works further in the project.